tensorflow/python/ops/nn_ops.py

```diff
@@ -519,7 +519,7 @@ def max_pool(value, ksize, strides, padding, data_format="NHWC", name=None):
     strides: A list of ints that has length >= 4.  The stride of the sliding
       window for each dimension of the input tensor.
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
-      See the [comment here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py)
+      See the [comment here](https://www.tensorflow.org/versions/r0.8/api_docs/python/nn.html#convolution)
```

Comment:
Just omit the /versions/r0.8 bit. It will redirect to the proper version
based on referrer.
On Wed, May 25, 2016 at 21:57 Vijay Vasudevan <notifications@github.com>
wrote:

> In tensorflow/python/ops/nn_ops.py
> <https://github.com/tensorflow/tensorflow/pull/2012#discussion_r64691671>:
>
> > @@ -519,7 +519,7 @@ def max_pool(value, ksize, strides, padding, data_format="NHWC", name=None):
> >      strides: A list of ints that has length >= 4.  The stride of the sliding
> >        window for each dimension of the input tensor.
> >      padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
> > -      See the [comment here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py)
> > +      See the [comment here](https://www.tensorflow.org/versions/r0.8/api_docs/python/nn.html#convolution)
>
> @martinwicke <https://github.com/martinwicke> do you remember how we can
> refer to the html documentation without hardcoding the version here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2012/files/168ac51ec34af1bd261a7b09ad70b248d66cbe5b..3be19d4359e5343b20bbbf43e1f1e75125e4d0b3#r64691671>
>